app/components/view_result_component.rb

```diff
@@ -0,0 +1,13 @@
+class ViewResultComponent < ViewComponent::Base
```

Comment:
Maybe `SearchResultCaptionComponent`?

Reply:
I have updated the component name as suggested